src/components/LayerCollection/LayerPO.js

```diff
@@ -8,8 +8,9 @@ import {
 } from '@folio/stripes/components';
 
 import { updateOrderResource } from '../Utils/orderResource';
-import { showUpdateOrderError } from '../Utils/order';
+import { showOpenOrderError } from '../Utils/order';
```

Comment:
do you think this name is correct? this function shows errors not only for open action, additionally you use `updateOrderErrorCode` above

Reply:
fixed